bees/filters.go

```diff
@@ -35,15 +37,24 @@ type Filter struct {
 }
 
 // execFilter executes a filter. Returns whether the filter passed or not.
-func execFilter(filter string, opts map[string]interface{}) bool {
-	f := *filters.GetFilter("template")
-	log.Println("\tExecuting filter:", filter)
+func execFilter(source string, opts map[string]interface{}) bool {
+	name := "template"
+	if strings.Contains(source, "def main(") {
```

Comment:
I'm not too happy with this "string magic". Maybe we could introduce proper filter types? I was also pondering having simple bash-scripts as filters, which either return 0 or 1 as a result.

Reply:
I'm also not sure if it is a good solution but it requires a selector on the UI side and I'm not ready to dive into Ember. If you're ready to do such adjustments, please, do.

If to talk about a better backend-only solution, I can propose shebangs. In that case, this line will be `if strings.HasPrefix(source, "# starlark") `, for example.

Bash scripts is a cool and fun idea. I'm sure some will be happy to build grep/sed/awk/perl pipelines.